src/control/lib/spdk/ctests/SConscript

```diff
@@ -0,0 +1,30 @@
+"""Build go-spdk bindings C tests"""
+from __future__ import print_function
```

Comment:
(pylint-misplaced-future) __future__ import is not the first non docstring statement
(pylint-reimported) Reimport 'print_function' (imported line 3)